src/lightgbm/src/main/scala/LightGBMBooster.scala

```diff
@@ -138,4 +139,13 @@ class LightGBMBooster(val model: String) extends Serializable {
     }
     new LightGBMBooster(model)
   }
+
+  def saveModel(session: SparkSession, filename: String): Unit = {
```

Comment:
the goal of this is to save the model in lightGBM Native format right? Perhaps call this something like saveNativeModel to distinguish from the spark model.